src/spec-common/variableSubstitution.ts

```diff
@@ -109,6 +110,9 @@ function replaceWithContext(isWindows: boolean, context: SubstitutionContext, ma
 		case 'containerWorkspaceFolderBasename':
 			return context.containerWorkspaceFolder !== undefined ? path.posix.basename(context.containerWorkspaceFolder) : match;
 
+		case 'remoteUser':
```

Comment:
I believe the behavior will now be that `devcontainer.json` configs will also replace `${remoteUser}`.  I think it would be good to add a test/confirm that behavior so we can document accordingly.

Reply:
Will do!